docker-compose.dist.yml

```
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  connect-documentation:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    ports:
+      - '${PORT}:8080'
+    volumes:
+      - '${APP_VOLUME}'
+    environment:
+      TZ: ${TZ-Europe/Amsterdam}
+    networks:
+      - default
+      - connect
+    labels:
+      - 'traefik.enable=true'
+      - 'traefik.http.routers.connect.tls=true'
+      - 'traefik.http.routers.connect.entrypoints=https'
+      - 'traefik.http.routers.connect.rule=Host(`connect.myofflinestore.com`)'
+
+networks:
+  default:
+    external:
+      name: gateway
+  connect:
+    internal: true
```

There's only 1 service so no need for an additional network
```suggestion
  connect:
    internal: true
```